docker-compose.yml

```diff
@@ -88,4 +95,6 @@ services:
       - 8005:8005
 
 volumes:
-  mongodb_vol:
\ No newline at end of file
+  mongodb_vol:
+  redis_data:
+    driver: local
```

Comment:
Why create volume for redis? Volumes are for data persistence and we don't need to keep the data once we terminate the localhost application for redis.